com.unity.cinemachine/Samples~/3D Samples/ClearShot.unity

```diff
@@ -0,0 +1,1608 @@
+%YAML 1.1
```

Comment:
Cameras in this scene start to break when you walk the character around.  Needs tuning or more restrictions on the movement.